src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1685,5 +1685,28 @@ sap.ui.define([
 		return this.bSecureSearch;
 	};
 
+	/**
+	 * Determines the next link url should be used
+	 *
```

Comment:
Add a comment:
```
/**
* Determines the next link url should be used
* 
* This method might be overridden by the application to provide a customized next link processing because FHIR did not offer a standardized link structure.
```

Reply:
done